app/controllers/users/invitations_controller.rb

```diff
@@ -59,58 +62,45 @@ def invite_users
       @invite_results = []
       @too_many_emails = false
 
-      cntr = 0
-      @emails.each do |email|
-        cntr += 1
-
-        if cntr > Constants::INVITE_USERS_LIMIT
+      @emails.each_with_index do |email, count|
```

Comment:
'count' is a bit misleading name in this method

Reply:
Fix